src/shapes/Group.ts

```diff
@@ -978,7 +980,7 @@ export class Group extends createCollectionMixin(
    */
   toObject(propertiesToInclude: (keyof this)[] = []) {
     return {
-      ...super.toObject([
+      ...this.toObject([
```

Comment:
ERROR

Reply:
Yes i have been trying to have this not complain and i tried different things, one was remove the toObject method from the base class